### PR TITLE
Add severity aliases

### PR DESCRIPTION
**Pull request by bazsi**
This PR adds support for setting the syslog severity based on alternative names that are produced by applications. For example scl/pgsql or the ones in #4760 

The ones we recognize are in lib/severity-aliases.table:

emerg,          SYSLOG_SEVERITY_CODE(0)
emergency,      SYSLOG_SEVERITY_CODE(0)
panic,          SYSLOG_SEVERITY_CODE(0)
alert,          SYSLOG_SEVERITY_CODE(1)
crit,           SYSLOG_SEVERITY_CODE(2)
critical,       SYSLOG_SEVERITY_CODE(2)
fatal,          SYSLOG_SEVERITY_CODE(2)
err,            SYSLOG_SEVERITY_CODE(3)
error,          SYSLOG_SEVERITY_CODE(3)
warning,        SYSLOG_SEVERITY_CODE(4)
warn,           SYSLOG_SEVERITY_CODE(4)
notice,         SYSLOG_SEVERITY_CODE(5)
info,           SYSLOG_SEVERITY_CODE(6)
log,            SYSLOG_SEVERITY_CODE(6)
debug,          SYSLOG_SEVERITY_CODE(7)

But it's almost free to populate more.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build FAILURE

File: scl/pgsql/pgsql.conf [hunk 1]
```diff
@@ -91,37 +91,19 @@ block parser postgresql-csvlog-parser(internal(yes) prefix(".pgsql.") on-type-er
             set(string("$(if ('${`prefix`connection_from}' ne '') ${`prefix`connection_from} $HOST_FROM)") value("`prefix`connection_from") internal(yes));
             set(string("${`prefix`message}") value("MESSAGE") internal(yes));
         };
-        # Map severity
-        if (match("DEBUG[1-5]" value('`prefix`severity') type(pcre))) {
+
+        # Map severity to match https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS
+	# for some reason PgSQL decided to map its internal levels in a
+        # shifted fashion above and including "error".
```
Comment (alltilla) [inline scl/pgsql/pgsql.conf:97]: PANIC should be shifted by 2:

| Severity | Usage | Syslog | eventlog |
| --- | --- | --- | --- |
| PANIC | Reports an error that caused all database sessions to abort. | CRIT | ERROR

```c
panic,		SYSLOG_SEVERITY_CODE(0)
crit,		SYSLOG_SEVERITY_CODE(2)
```

**Review by alltilla — COMMENTED** [review]
One small comment, otherwise approve.

**Comment by alltilla** [comment]
Pushed one commit with a packaging related news entry.

File: scl/pgsql/pgsql.conf (hunk 1 above)
Comment (bazsi) [reply scl/pgsql/pgsql.conf]: good catch. thanks. I added a testcase and a fix.

**Review by alltilla — DISMISSED** [review]

**Review by alltilla — APPROVED** [review]

**Comment by alltilla** [comment]
Test CI run with a manually built devshell: https://github.com/alltilla/syslog-ng/actions/runs/7397098190

**Comment by alltilla** [comment]
Copyright check fail only, we should fix that and merge this with red CI:
```
error: missing license lib/severity-aliases.table (nonstandard license declaration?)
```

**Review by alltilla — APPROVED** [review]

**Pull request merged**